docs/nodes/nodes-library/nodes/CoinGecko/README.md

```diff
@@ -0,0 +1,111 @@
+---
+permalink: /nodes/n8n-nodes-base.coinGecko
+description: Learn how to use the CoinGecko node in n8n
+---
+
+# CoinGecko
+
+[CoinGecko](https://www.coingecko.com) provides a fundamental analysis of the crypto market. In addition to tracking price, volume and market capitalization, CoinGecko tracks community growth, open-source code development, major events and on-chain metrics.
+
+::: tip 🔑 Credentials
+The CoinGecko node does not require authentication.
+:::
+
+## Basic Operations
+
+::: details Coin
+- Get a candlestick open-high-low-close chart for the selected currency
+- Get current data for a coin
+- Get all coins
+- Get historical data (name, price, market, stats) at a given date for a coin
+- Get prices and market-related data for all trading pairs that match the selected currency
+- Get historical market data that includes the price, market cap, and 24 hours volume (granularity auto)
+- Get the current price of any cryptocurrencies in any other supported currencies that you need.
+- Get coin tickers
+:::
+
+::: details Event
+- Get all events
+:::
+
+
+## Example Usage
+
+This workflow allows you to get the price of BTC in EUR using the CoinGecko node and send an SMS when the price is larger than or equal to EUR 9000. You can also find the [workflow](https://n8n.io/workflows/704) on n8n.io. This example usage workflow would use the following nodes.
+- [Cron](../../core-nodes/Cron/README.md)
+- [CoinGecko]()
+- [IF](../../core-nodes/If/README.md)
+- [Twilio](../../nodes/Twilio/README.md)
+- [No Operation, do nothing](../../core-nodes/NoOperationDoNothing/README.md)
+
+The final workflow should look like the following image.
+
+![A workflow with the CoinGecko node](./workflow.png)
+
+### 1. Cron node
+
+The Cron node will trigger the workflow every hour.
+
+1. Click on ***Add Cron Time***.
+2. Select 'Every Hour' from the ***Mode*** dropdown list.
```

Comment:
This should probably run every minute. Screenshots and workflow will have to be adjusted too.